### ISSUE TITLE
Filesystem

### ISSUE THREAD

**Issue by cryptosbyte** [PenetratingShot/flame#3]
Heya, I saw you on the [os-tutorial github issues repo](https://github.com/cfenollosa/os-tutorial/) and was willing if you can contact me to help me make a file system and a proper fixed shell, just like yours; I know yours is opensource but I'd seem that you explaining to me would be helpful; hopefully this reaches you soon and can you kindly friend me on Discord so we can chat some time, thanks; Cryptos#9754.

TL;DR I have left off the lesson 23 and want you to help me make a file system like yours; can you contact me on Discord from the above. Thanks.

**Comment by shreyaslad** [comment]
https://discord.gg/RnCtsqD
